include/simdjson/arm64.h

```diff
@@ -7,6 +7,11 @@
 
 #include "simdjson/portability.h"
 
+#ifndef SIMDJSON_IMPLEMENTATION_ARM64
+#define SIMDJSON_IMPLEMENTATION_ARM64 (SIMDJSON_IS_ARM64)
+#define SIMDJSON_CAN_ALWAYS_RUN_ARM64 (SIMDJSON_IS_ARM64)
```

Comment:
The other ones are doing this outside the ifndef, in case the user passes it in:

```c++
#ifndef SIMDJSON_IMPLEMENTATION_ARM64
#define SIMDJSON_IMPLEMENTATION_ARM64 (SIMDJSON_IS_ARM64)
#endif
#define SIMDJSON_CAN_ALWAYS_RUN_ARM64 SIMDJSON_IMPLEMENTATION_ARM64 && SIMDJSON_IS_ARM64
```

Reply:
Will correct.